rpm.mk

```diff
@@ -47,32 +47,35 @@ bundle-rust:
 
 install-node-modules:
 ifeq ($(COCKPIT_ON), 1)
-	cd src/cockpit/389-console; make -f node_modules.mk install
+	cd src/cockpit/389-console; \
+	rm -rf node_modules; \
+	rm -rf node_modules.release; \
+	npm ci > /dev/null
 endif
 
 build-cockpit: install-node-modules
 ifeq ($(COCKPIT_ON), 1)
+ifndef SKIP_AUDIT_CI
+	cd src/cockpit/389-console; \
+	npx audit-ci
+endif
 	cd src/cockpit/389-console; \
 	rm -rf cockpit_dist; \
-	NODE_ENV=production make -f node_modules.mk build-cockpit-plugin
+	NODE_ENV=production ./build.js; \
+	cp -r dist cockpit_dist
 endif
 
 dist-bz2: install-node-modules download-cargo-dependencies
 ifeq ($(COCKPIT_ON), 1)
 	cd src/cockpit/389-console; \
 	rm -rf cockpit_dist; \
-	NODE_ENV=production make -f node_modules.mk build-cockpit-plugin; \
```

Comment:
Previously, we were running audit-ci as part of `NODE_ENV=production make -f node_modules.mk build-cockpit-plugin;` here at `dist-bz2:`. But now when I execute `TAG=389-ds-base-2.4.4 make -f rpm.mk dist-bz2` - audit-ci is not run at all.

Should we also add it here?